token-client/src/main/java/com/sap/cloud/security/xsuaa/client/OAuth2TokenKeyService.java

```diff
@@ -16,31 +16,48 @@
 public interface OAuth2TokenKeyService {
 
 	/**
-	 * Same as {@link #retrieveTokenKeys(URI, String)} except that zoneId is set to
-	 * {@code null}.
-	 *
 	 * @deprecated gets removed in favor of {@link #retrieveTokenKeys(URI, String)}
 	 *             with next major version 3.0.0
 	 */
 	@Deprecated
 	default String retrieveTokenKeys(@Nonnull URI tokenKeysEndpointUri) throws OAuth2ServiceException {
 		throw new UnsupportedOperationException(
-				"for security reason this internal method was replaced with retrieveTokenKeys(<jwks_url>, <zone_uuid>)");
+				"for security reason this internal method was replaced with retrieveTokenKeys(<jwks_url>, <app_tid>)");
 	}
 
 	/**
-	 * Requests token web key set from OAuth Server.
+	 * Requests token web key set from Xsuaa OAuth Server.
+	 *
+	 * @param tokenKeysEndpointUri
+	 *            the token endpoint URI (jku).
+	 * @param tenantId
+	 *            the tenant id of the tenant. Obligatory parameter in context of
+	 *            multi-tenant IAS applications to make sure that the tenant id
```

Comment:
Here we suddenly talk about IAS only. What should go into tenantId in case of XSUAA? null or zoneId?